Code/server/src/main.cpp

```
@@ -56,6 +56,8 @@ int main(int argc, char** argv)
         }
 
         GameServer server(port, premium, name.c_str(), token.c_str());
+        if (!server.Initialize())
+            return -1;
```

Might as well just throw a std::runtime_exception in Initialize instead of returning a bool